### PR TITLE
Add grovetccfg no-service-reload arg, document error codes.

### PR DESCRIPTION
**Pull request by rob05c**
Add grovetccfg no-service-reload arg.

Also changes error codes to an enum, so code is self-documenting,
and adds readme documentation of error codes.

#### What does this PR do?

Fixes #(issue_number)

#### Which TC components are affected by this PR?

- [ ] Documentation
- [x] Grove
- [ ] Traffic Analytics
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] Other _________

#### What is the best way to verify this PR?

Run 
```
cd trafficcontrol/grove/grovetccfg
go build
mkdir testcertdir

./grovetccfg -tourl https://my-to-host.example.net -host edge-cache-hostname -touser myUser -topass myPass -ignore-update-flag -insecure -pretty -certdir testcertdir
```
On a machine with no Grove service running. Should fail with exit code 2, when it can't restart the service.

Run again with `-no-service-reload` argument, should succeed with exit code 0.

#### Check all that apply

- [ ] This PR includes tests
- [ ] This PR includes documentation updates
- [ ] This PR includes an update to CHANGELOG.md
- [ ] This PR includes all required license headers
- [ ] This PR includes a database migration (ensure that migration sequence is correct)
- [ ] This PR fixes a serious security flaw. Read more: [www.apache.org/security](http://www.apache.org/security/)

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3026/
Test PASSed.

**Comment by jrushford** [comment]
Looks good @rob05c.  Thanks for working on this!

File: grove/grovetccfg/grovetccfg.go [hunk 1]
```diff
@@ -43,9 +43,6 @@ import (
 	"github.com/apache/trafficcontrol/grove/web"
 )
 
-// Duplicating Hdr and ModHdrs here for now...
-// Seems cleaner than dragging it up from some arbitrary place in plugins
-
 const Version = "0.1"
```
Comment (jrushford) [inline grove/grovetccfg/grovetccfg.go]: @rob05c could you change the version to "0.2".  I changed the rpm build scripts and grove to "0.2" last November and missed changing it in grovetccfg.  the rpm build script for grovetccfg is using version "0.2".

Comment (rob05c) [reply grove/grovetccfg/grovetccfg.go]: Done

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/trafficcontrol-PR/3091/
Test PASSed.

**Pull request merged**